README.md

```diff
@@ -54,6 +58,78 @@ mutation {
 }
 ```
 
+### Scoped documents for a signatory
+
+```
+mutation {
+  addSignatory(
+    input: {
+      signatureOrderId: "[signatureOrder.id]"
+      documents: [
+        {id: "[signatureOrder.documents[0].id]"}
+      ]
+    }
+  ) {
+    signatory {
+      href
+    }
+  }
+}
+```
+
+*Same input format is available for workflow participants*
+
+### Preapproved documents for a signatory
+
+By preapproving documents for a signatory they are immediately sent to e-ID login to sign and skip the frontend approval flow.
+
+When using preapproval document-scoping is automatically used so make sure that you pass all documents from the signature order to the signatory.
+
+```
+mutation {
+  addSignatory(
+    input: {
+      signatureOrderId: "[signatureOrder.id]"
+      documents: [
+        {id: "[signatureOrder.documents[0].id]", preapproved: true}
```

Comment:
How would this work when one wants to include all the documents (so not hardcoding the indices)

Reply:
@sgryt You would simply omit the `documents` argument. Or if preapproving all documents you'd need a map/loop in the client code.

Reply:
> @sgryt You would simply omit the `documents` argument. Or if preapproving all documents you'd need a map/loop in the client code.

Oh, I did not know that the `graphql` construct supported loops. Good to know :)

Reply:
@sgryt Well the query itself does not support loops, but you can use an input variable like `documents: $documents` and your code can then use a loop to generate that variable.